src/index.css

```diff
@@ -1,13 +1,41 @@
+@import url('styles/utils.css');
+
+h1, h3, p {
+    margin: 0;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
-  -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+#root {
+    height: 100vh;
+    overflow-y: scroll;
+    margin: 0 auto;
```

Comment:
Check if this is needed

Reply:
It felt necessary for the sake of the gradient background... although the margin prop had no effect, so I got rid of it.